src/+otp/+bouncingball/onEvent.m

```diff
@@ -5,8 +5,9 @@
 % horizontal plane, we then invert the y velocity, and transform
 % back.
 
-pos = sol.ye(1:2, end);
-vel = sol.ye(3:4, end);
+% OCTAVE BUG: event data is transposed compared to MATLAB
```

Comment:
Is it possible to do some type of try catch thing to do different stuff for matlab and octave here so that everything works?

Reply:
The reshape handles things correctly for both MATLAB and Octave. I'll label this as `OCTAVE FIX` instead.